src/GeoJSON.Text.Tests/GeoJSON.Text.Tests.csproj

```diff
@@ -2,17 +2,68 @@
   <PropertyGroup>
     
     <ProjectGuid>{6C93B314-9208-4684-B873-172F7EC81689}</ProjectGuid>
-    <RootNamespace>GeoJSON.Net.Tests</RootNamespace>
-    <AssemblyName>GeoJSON.Net.Tests</AssemblyName>
-    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <RootNamespace>GeoJSON.Text.Tests</RootNamespace>
+    <AssemblyName>GeoJSON.Text.Tests</AssemblyName>
+    <TargetFrameworks>netcoreapp3.1;net5;net6</TargetFrameworks>
```

Comment:
`TargetFrameworks` we should mimic what is supported by `System.Text.Json`.

It would be `<TargetFrameworks>netcoreapp3.1;netstandard2.0;net461;net6</TargetFrameworks>`

Reply:
Very valid point.
As far as I can see the supported frameworks are.

**Build into the framework itself**
.NET Core 3.0 and onwards.

**If installed as a nuget package**
.NET Standard 2.0 and later versions
.NET Framework 4.7.2 and later versions
.NET Core 2.0, 2.1, and 2.2

Reply:
For me it's not about what frameworks Microsoft is supporting, it's what frameworks people are using. If you look at the binaries in the `System.Text.Json` package they build a binary for `net461` so in essence, they are supporting that version and people are able to use `System.Text.Json` so we shouldn't exclude them.

Net461 is still in support until April [Microsoft Net Framework Lifecycle](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework)

Reply:
Well.. supporting older versions can result in the fact, that there are several .NET improvements you can't use, and performance improvements that are unavailable to you. Which can make sense of course.
But I think this is a bit different, since system.text.json is relatively new, and I think that most people will not be, and is not already using this package in the older versions of .NET, since the standard use in these frameworks are Newtonsoft.Net.

It just seems like a lot of additional work to cut yourself off the newer versions of .NET to support the last 0,4 percent of the users of System.Text.Json?

Wouldn't it make more sense to support .NET core and .NET going forward as the primary target of this package?

Reply:
ok leave it for now